src/app/components/toolbar/toolbar.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Remove the spec.ts files, because we don't use them at this point.

Reply:
So i should remove the completely from the project?

Reply:
You can also modify the .gitignore file to ignore this type of files